README.md

```diff
@@ -463,3 +463,20 @@ cmake -DCMAKE_BUILD_TYPE=Release -DBENCHMARK_ENABLE_LTO=true
 
 ## Linking against the library
 When using gcc, it is necessary to link against pthread to avoid runtime exceptions. This is due to how gcc implements std::thread. See [issue #67](https://github.com/google/benchmark/issues/67) for more details.
+
+
+## Running a subset of the benchmarks
+
+Similar to the `--gtest_filter` functionality of `googletest`, `benchmark` has a `--benchmark_filter` option:
```

Comment:
I don't think anyone cares what sort of regex the filter takes; seems a little detailed for a README. But nonetheless, if the analogy is confusing, why don't we just write:

> To run a subset of the benchmarks, use the `--benchmark-filter` option:

and forget about googletest?